torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1025,6 +1051,92 @@ def determine_observer_insert_points(self, prepared_fx_model: GraphModule) -> Di
 
         return obs_fqn_to_info
 
+    def _calculate_outlier_info(self, percentile_ratios: torch.Tensor, counted_batches: torch.Tensor) -> Dict[str, List[bool]]:
+        r"""
+        Gives info on whether the percentile ratios cacluated would be considered outliers
+        Also gives information on whether the collected data is statistically significant to make this claim
+
+        Args:
+            percentile_ratios (torch.Tensor): The average percentile_ratios per channel calculated by the observer
+            counted_batches (torch.Tensor): The number of batches used for average calculation per tensor
+
+        Returns a dictionary mapping:
+            "outliers_detected" : list of bools per channel that are true if it is considered an outlier
+            "statistically_significant": if the per channel calculation had at least 30 samples
```

Comment:
again the language is a bit suspect

Reply:
changed to say 

`"above_sample_threshold_count": if the per channel calculation had at least 30 samples (a random threshold)`

Reply:
will update when i do the threshold calculation and make it a parameter, so this will be updated soon in a coming PR